resources/env/test/phpunit/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM php:7.2-cli
```

Comment:
Can we do that directly using the composer image? I'm pretty sure we can overload the existing CMD